runelite-client/src/main/java/net/runelite/client/plugins/raids/RaidsPlugin.java

```diff
@@ -807,4 +907,50 @@ private void screenshotScoutOverlay()
 		imageCapture.takeScreenshot(overlayImage, "CoX_scout-", false, config.uploadScreenshot());
 		graphic.dispose();
 	}
+
+	private int findLobbyIndex(Point gridBase)
```

Comment:
It would be lovely to have some documentation on this, or at least constants relating to the otherwise magic numbers in this method. (why is `y` only 0 or 1? why is `x`'s max value 3?)